### PR TITLE
evdi: unstable-20221013 -> 1.13.1

### PR DESCRIPTION
**Pull request by jensbin**
###### Description of changes

- Pull request to update evdi to 1.13.1
- Needed to use with Linux kernel >= 6.2

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by Janik-Haag — COMMENTED** [review]
I'm not familiar with how to test this package, we will have to wait until @eyJhb does so.
Code  looks good to me.

File: pkgs/os-specific/linux/evdi/default.nix [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
-  version = "unstable-2022-10-13";
+  version = "1.13.1";
 
   src = fetchFromGitHub {
     owner = "DisplayLink";
     repo = pname;
-    rev = "bdc258b25df4d00f222fde0e3c5003bf88ef17b5";
-    hash = "sha256-mt+vEp9FFf7smmE2PzuH/3EYl7h89RBN1zTVvv2qJ/o=";
+    rev = "v${version}";
+    sha256 = "sha256-Or4hhnFOtC8vmB4kFUHbFHn2wg/NsUMY3d2Tiea6YbY=";
```
Comment (drupol) [inline pkgs/os-specific/linux/evdi/default.nix:15]:
```suggestion
    hash = "sha256-Or4hhnFOtC8vmB4kFUHbFHn2wg/NsUMY3d2Tiea6YbY=";
```

**Review by drupol — CHANGES_REQUESTED** [review]
LGTM, just a minor change to apply.

Regarding for testing this package, I'll pass it to someone else as well.

**Comment by jensbin** [comment]
Verified with:
- HP G5 Universal Dock
- HP Thunderbolt Dock G4

**Comment by Janik-Haag** [comment]
Can you provide instructions on how to verify that I have access to usb c/thunderbolt docs and a laptop that supports those, just don't know how to test this package?

**Comment by drupol** [comment]
Shouldn't we be using version 1.14.0 ? https://github.com/DisplayLink/evdi/tags

**Comment by jensbin** [comment]
Very high-level: Attaching to a displaylink docking station w/o `evdi` (this package) and `displaylink` (the other package) does not show any output. Because of missing proprietary displaylink software.

Instruction how to enable display on nixos are on that wiki: https://nixos.wiki/wiki/Displaylink

With these instructions the monitor signal is getting through the displaylink of the dock.

There are a couple of changes listed in evdi and displaylink changelog, the most obvious is that the version <1.13.1 does not compile with kernels >6.1

**Comment by jensbin** [comment]
> Shouldn't we be using version 1.14.0 ? https://github.com/DisplayLink/evdi/tags

The current supported and bundled version by the needed displaylink daemon is 1.13.1.
1.14.0 might be supported by the next displaylink software.

**Comment by drupol** [comment]
Ok, fair enough.

could you please fix the commit log message to match the PR title and I'll merge this.

**Review by drupol — APPROVED** [review]

**Pull request merged**